### PR TITLE
Add workflow methods createConfigured and export

### LINKED ISSUE #28: Add workflow methods createConfigured and export

### PR DESCRIPTION
**Pull request by simonduerr**
Fixes #28.

### CODE REVIEW

File: src/Hydrator/Config/Workflow/WorkflowSchemaVersionEnum.php [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?php declare(strict_types=1);
```
Comment (WorksDev) [inline src/Hydrator/Config/Workflow/WorkflowSchemaVersionEnum.php]: At the moment this makes no sense for me - the version number is always indicated in the client itself.

If there is a new API version, there must also be a new client, with new methods and a new version number.

File: tests/Client/Workflow/WorkflowClientTest.php [hunk 2]
```diff
@@ -7,6 +7,7 @@
 use Scn\EvalancheSoapApiConnector\Extractor\ExtractorInterface;
 use Scn\EvalancheSoapApiConnector\Hydrator\Config\HydratorConfigFactoryInterface;
 use Scn\EvalancheSoapApiConnector\Hydrator\Config\HydratorConfigInterface;
+use Scn\EvalancheSoapApiConnector\Hydrator\Config\Workflow\WorkflowSchemaVersionEnum;
```
Comment (WorksDev) [inline tests/Client/Workflow/WorkflowClientTest.php]: Does no longer exist

File: src/Client/Workflow/WorkflowClient.php [hunk 3]
```diff
@@ -101,4 +101,44 @@ public function pushProfilesIntoCampaign(int $id, array $profileIds): bool
...
+
+    /**
+     * Export a specific workflow configuration as JSON string
+     *
+     * @param string $workflowId
+     *
+     * @return string
+     * @throws EmptyResultException
+     */
+    public function export(string $workflowId): string
```
Comment (WorksDev) [inline src/Client/Workflow/WorkflowClient.php]: Test seems to be missing

**Comment by usox** [comment]
@WorksDev I think, we're done here?

**Pull request merged**